webui/react/src/services/api.ts

```diff
@@ -7,8 +7,6 @@ import { generateDetApi } from 'shared/utils/service';
 import * as Type from 'types';
 import { tensorBoardMatchesSource } from 'utils/task';
 
-export { isLoginFailure } from './utils';
```

Comment:
avoid having to re-export, just import directly from `services/utils`

Reply:
👍